### PR TITLE
[utils] exact check node ip address

### PR DESCRIPTION
**Pull request by JiahaoYao**
maybe ci not working?

### CODE REVIEW

**Comment by JiahaoYao** [comment]
@zhuohan123

File: alpa/util.py [hunk 1]
```diff
@@ -1109,3 +1110,9 @@ def wrapper(*args, **kwargs):
     except ImportError:
         logger.warning("Install numba to jit and accelerate the function.")
         return func
+
+
+def is_host(resource_key):
```
Comment (zhuohan123) [inline alpa/util.py:1115]: Maybe rename to `is_ray_node_resource`?

**Review by zhuohan123 — APPROVED** [review]
LGTM. Suggested a name change

File: alpa/util.py (hunk 1 above)
Comment (JiahaoYao) [reply alpa/util.py]: merged

**Comment by JiahaoYao** [comment]
```python
test_dot_reshape_transpose (test_auto_sharding_basic.AutoShardingBasicTest) ... 2022-06-17 00:49:05.171032: E external/org_tensorflow/tensorflow/stream_executor/cuda/cuda_blas.cc:232] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2022-06-17 00:49:05.171118: E external/org_tensorflow/tensorflow/stream_executor/cuda/cuda_blas.cc:234] Failure to initialize cublas may be due to OOM (cublas needs some free memory when you initialize it, and your deep-learning framework may have preallocated more than its fair share), or may be because this binary was not built with support for the GPU in your machine.
2022-06-17 00:49:05.171188: E external/org_tensorflow/tensorflow/compiler/xla/pjrt/pjrt_stream_executor_client.cc:2143] Execution of replica 0 failed: INTERNAL: Attempting to perform BLAS operation using StreamExecutor without BLAS support
ERROR
```

test breaks

**Comment by merrymercy** [comment]
Sometimes CI is not very stable. You can try to re-trigger it.

**Pull request merged**